### PR TITLE
Feature-7895 create catalog file part 2

### PR DESCRIPTION
**Pull request by shailen14599**
Code changes
Unit tests
Common changes for readme unit tests

### CODE REVIEW

File: UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentAncillaryFiles.cs [hunk 1]
```diff
@@ -169,5 +200,30 @@ public bool CreateProductFile(string batchId, string exchangeSetInfoPath, string
...
             }
         }
+
+        private string CrcString(string fullFilePath)
+        {
+            var crcHash = Crc32CheckSumProvider.Instance.Compute(fileSystemHelper.ReadAllBytes(fullFilePath));
+            return crcHash.ToString();
+        }
+
+        private string GeIsdtUadt(SalesCatalogueDataProductResponse salescatalogProduct)
```
Comment (janvi13603) [inline UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentAncillaryFiles.cs:210]: Kindly change method name to GetIssueAndUpdateDate

File: UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/FileSystemHelper.cs [hunk 2]
```diff
@@ -89,5 +89,51 @@ public bool CheckFileExists(string filePath)
         {
             return File.Exists(filePath);
         }
+
+        public byte[] ReadAllBytes(string filePath)
+        {
+            return File.ReadAllBytes(filePath);
```
Comment (janvi13603) [inline UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/FileSystemHelper.cs:95]: Can we check if fileexists before reading bytes data

File: UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentAncillaryFiles.cs (hunk 1 above)
Comment (shailen14599) [reply UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentAncillaryFiles.cs]: Resolved

File: UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/FileSystemHelper.cs (hunk 2 above)
Comment (shailen14599) [reply UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/FileSystemHelper.cs]: Modified. Resolved.

**Review by janvi13603 — APPROVED** [review]
Changes looks good.

**Review by ChrisEllen2020 — APPROVED** [review]
Looks good

**Pull request merged**